GoogleCloud/README.md

```diff
@@ -57,13 +57,15 @@ Included is a tutorial in the form of Jupyter notebooks. The main purpose of the
+**2)** Follow the steps highlighted [here](https://github.com/NIGMS/NIGMS-Sandbox/blob/main/docs/HowToCreateVertexAINotebooks.md) to create a new instance notebook in Vertex AI. Follow steps 1-8 and be especially careful to enable idle shutdown as highlighted in [step 8](https://github.com/NIGMS/NIGMS-Sandbox/blob/main/docs/HowToCreateVertexAINotebooks.md#:~:text=On%20the%20same%20screen%2C%20verify%20that%20Enable%20Idle%20Shutdown%20is%20selected%20and%20specify%20the%20idle%20minutes%20for%20shutdown.%20This%20means%20it%20will%20shutdown%20automatically%20after%20this%20many%20minutes.%20We%20recommend%2030%20minutes.). For this module you should select Debian 11 and Python 3 in the Environment tab in [step 6](https://github.com/NIGMS/NIGMS-Sandbox/blob/main/docs/HowToCreateVertexAINotebooks.md#:~:text=On%20the%20Environment%20screen). In [step 7](https://github.com/NIGMS/NIGMS-Sandbox/blob/main/docs/HowToCreateVertexAINotebooks.md#:~:text=On%20the%20Machine%20type%20screen) in the Machine type tab, select n1-standard-4 from the dropdown box.
 
 **3)** Now you will need to download the tutorial files from GitHub. The easiest way to do this would be to clone the repository from NIGMS into your Vertex AI notebook. This can be done by using the `Git` menu in JupyterLab, and selecting the clone option. To clone this repository, use the Git command `git clone https://github.com/NIGMS/Introduction-to-Data-Science-for-Biology.git` in the dropdown menu option in Jupyter notebook. Please make sure you only enter the link for the repository that you want to clone. There are other bioinformatics related learning modules available in the [NIGMS Repository](https://github.com/NIGMS). This will download our tutorial files into a folder called `Introduction-to-Data-Science-for-Biology`.
 
+**3.1)** (Alternative Method) If any menus do not work as described in step 3 above, this is an alternative method to achieve the same result: [New Terminal How-to](https://github.com/NIGMS/NIGMS-Sandbox/blob/main/docs/HowToCreateNewTerminalConsoleJupyterLab.md)
```

Comment:
Change to **3.1)** (Alternative Method) If any menus do not work as described in step 3 above, use [this method to open a new terminal](https://github.com/NIGMS/NIGMS-Sandbox/blob/main/docs/HowToCreateNewTerminalConsoleJupyterLab.md), then clone the repo with `git clone https://github.com/NIGMS/Introduction-to-Data-Science-for-Biology.git`.